kotlinx-coroutines-core/jvmBenchmark/kotlin/kotlinx/coroutines/SemaphoreBenchmark.kt

```diff
@@ -81,7 +80,6 @@ open class SemaphoreBenchmark {
 
 enum class SemaphoreBenchDispatcherCreator(val create: (parallelism: Int) -> CoroutineDispatcher) {
     //    FORK_JOIN({ parallelism -> ForkJoinPool(parallelism).asCoroutineDispatcher() }),
```

Comment:
Why is this not uncommented, but similar code in `ChannelProducerConsumerBenchmark.kt` is?

Reply:
No real reason, uncommented as well